jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -75,37 +77,59 @@ function configure_router_state() {
 }
 
 function configure_router_location {
-    # DeploymentConfig environment
-    #
-    # name: KIE_SERVER_ROUTER_HOST
-    # valueFrom:
-    #   fieldRef:
-    #     fieldPath: status.podIP
-    #
-    local kieServerRouterHost="${KIE_SERVER_ROUTER_HOST}"
-    if [ "${kieServerRouterHost}" = "" ]; then
-        kieServerRouterHost="${HOSTNAME}"
-        if [ "${kieServerRouterHost}" = "" ]; then
-            kieServerRouterHost="localhost"
+
+    local routeName="${KIE_SERVER_ROUTER_NAME}"
+    local host="${KIE_SERVER_ROUTER_HOST}"
+    local port="${KIE_SERVER_ROUTER_PORT}"
+    local protocol="${KIE_SERVER_ROUTER_PROTOCOL}"
```

Comment:
Why were these script variables renamed? It was more clear from the original code as they matched the environment variable names. It doesn't hurt anything renaming them, but seems unnecessary.

Reply:
+1, why rename it?

Reply:
I renamed these variables to leave standardized according to this function: https://github.com/jboss-container-images/jboss-kie-modules/blob/master/jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh#L354